tools/getkeys/.eslintrc.js

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+    "extends": [
+        require.resolve("@fluidframework/eslint-config-fluid")
+    ],
+    "parserOptions": {
+        project: ["./tsconfig.json"],
+    },
+<<<<<<< HEAD
+    "env": {
+        "browser": true,
+        "node": true
+    },
+=======
+>>>>>>> 15109ae46a344f09fb99c2b1bd63923df50cdc02
+    "rules": {
```

Comment:
Looks like you have some merge markers in this file, maybe from an earlier incomplete merge?

Reply:
Ohh. I through conflicts should be resolved with `git rebase -i`. Making another commit to resolve it. Thanks for catching up.